jenkins/internal-ci/generic/branch/cortx-py-utils.groovy

```diff
@@ -89,7 +89,7 @@ pipeline {
                     def releaseBuild = build job: 'Release', propagate: true
                      env.release_build = releaseBuild.number
                     env.release_build_location = "http://cortx-storage.colo.seagate.com/releases/cortx/github/$branch/$os_version/${env.release_build}"
-                    env.cortx_images = releaseBuild.buildVariables.cortx_all_image +"\n" +releaseBuild.buildVariables.cortx_rgw_image +"\n" +releaseBuild.buildVariables.cortx_data_image +"\n" +releaseBuild.buildVariables.cortx_control_image
```

Comment:
Please validate that cortx-py-utils package creation is working with changes

Reply:
1) Checked and updated the applicable fields in PR Checklist template 

2) py-util is working fine
https://eos-jenkins.colo.seagate.com/job/Cortx-PR-Build/job/Cortx-Deployment/job/Generic/job/py-utils/742/

2) Validation on cortx-py-utils is in progress once its done will update Jenkins pipeline details